### ISSUE TITLE
Delayed ability to crawl 

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5883]
**Preferred term label:** Delayed ability to crawl

**Synonyms**

**Definition (free text, please give PubMed ID)** A failure to achieve the ability to crawl at an appropriate developmental stage. 

**Parent term (use hpo.jax.org/app)** Delayed gross motor development HP:0002194

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033128
